src/types/onyx/NewGroupChatDraft.ts

```diff
@@ -4,8 +4,9 @@ type SelectedParticipant = {
 };
 
 type NewGroupChatDraft = {
-    participants: SelectedParticipant[];
-    reportName: string;
+    participants?: SelectedParticipant[] | null;
```

Comment:
*suggestion:* rather than making `participants` optional, another way to do it would be to make `Report.setGroupDraft` take a `Parial<NewGroupChatDraft>`, which kind of makes sense for the merge case and keeps `participants` required/intact in other cases

Reply:
I'll try it !